elementary/monitor/dbt_project/macros/get_test_rows_sample.sql

```diff
@@ -1,13 +1,31 @@
-{%- macro get_test_rows_sample(result_rows_agate, test_type, results_sample_limit=5) -%}
-    {% set test_rows_sample = [] %}
+{%- macro get_test_rows_sample(result_rows, result_rows_agate, test_type, test_query, results_sample_limit=5) -%}
```

Comment:
The parameter names here are a bit confusing - not sure someone stumbling on this code will immediately understand what's the difference between `result_rows` and `result_rows_agate`.

Maybe the first should be something like `result_rows_from_json` or `legacy_result_rows`?
Either way I'd add a comment clarifying it.